package.json

```diff
@@ -30,6 +30,6 @@
     "uuid": "3.0.1"
   },
   "engines": {
-    "node": ">=6.4.0"
+    "node": "18.x"
```

Comment:
Ik weet het even niet meer. Was dit nou nodig? Of was alleen de specificatie in de highcharts-export-server repo genoeg? Want dan zou ik het hier weghalen, zodat je het niet op twee plekken hoeft te updaten.